Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -1487,30 +1492,27 @@ protected void onPostExecute(ObjectCursor<Note> cursor) {
         }
     }
 
-    private static class pinNotesTask extends AsyncTask<Void, Void, Void> {
-
-        private SoftReference<NoteListFragment> fragmentRef;
+    private static class PinNotesTask extends AsyncTask<Void, Void, Void> {
+        private SoftReference<NoteListFragment> mNoteListFragmentReference;
         private SparseBooleanArray mSelectedRows = new SparseBooleanArray();
 
-        private pinNotesTask(NoteListFragment context) {
-            fragmentRef = new SoftReference<>(context);
+        private PinNotesTask(NoteListFragment context) {
+            mNoteListFragmentReference = new SoftReference<>(context);
         }
 
         @Override
         protected void onPreExecute() {
-            NoteListFragment fragment = fragmentRef.get();
-            if (fragment.getListView() != null) {
-                mSelectedRows = fragment.getListView().getCheckedItemPositions();
-            }
+            NoteListFragment fragment = mNoteListFragmentReference.get();
+            mSelectedRows = fragment.getListView().getCheckedItemPositions();
```

Comment:
Is there a reason the null check is removed?

Reply:
It is unnecessary since `fragment.getListView()` is never `null`.